index.html

```diff
@@ -23,14 +23,18 @@ <h1 align="center">Mega Todo</h1>
         </tr>
         <tr id="baseRow">
             <td id="assignmentCell"><input class="assignmentName" type="text" /></td> <!-- Input -->
-            <td> 
-                <input list="Classes" type="text" name="class section"  class="className">
-                <datalist id="Classes">
+
+            <td>
+                <input list="citynames" name="cityname" id="cityname" class="className">
+                <datalist id="citynames" class="DataList">
```

Comment:
Semantically better. Commit this along with the below Javascript commit. 
```suggestion
                <input list="classes" class="className">
                <datalist id="classes" class="DataList">
```